portfolio/src/main/webapp/about.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>About</title>
+    <link rel="stylesheet" href="about.css">
+    <script src="truths_and_lies.js"></script>
+  </head>
+  <body>
+    <div class="about_content">
+    <div id="header">
+        <a href="index.html"> <img src=/images/profile.jpg /> </a>
+        <div class="left">
+            <a href="#"> <p> About </p> </a>
+            <a href="survey.html"> <p> Survey </p> </a>
+            <a href="contact.html"> <p> Contact </p> </a>
+        </div>
+    </div>
+    <div id="about">
+        <h3>Want to get to know me? Let's play Two Truths One Lie</h3>
+        <div id="truth_and_lies_game">
+            <label id="question_num">Press Start to Begin!</label>
+            <div id="questions">
+
+                <!-- These fields will be populated by the about.js file -->
+
+                <div class="truth_and_lies_question">
+                    <label>
+                        <input type="radio" name="chosen_answer" id="first_answer" value="0">
```

Comment:
Use a hyphen instead of an underscore for the id: first-answer.

As for the "name" attribute... Google's style guide specifically requires hyphens for the class and ID fields but says nothing about the name field.  For consistent reading, I would recommend going with "chosen-answer", using a hyphen as well (it's what I would do in my own code, at least).

These comments also apply to the other two answer divs below =)

Reply:
changed to hyphens